products/extensions/b2b-components/index.md

```diff
@@ -20,3 +20,134 @@ In the world of digital B2B commerce, where businesses engage with other compani
 * **Quick Order and Shopping List** takes care of distinctive B2B buying behaviors.
 
 * **Digital Sales Composables** aims to provide a set of composable frontends to cover more complex Sales Representative jobs.
+
+## Configuring custom toggles for B2B components
+
+The B2B Components allow merchants to selectively choose and configure B2B capabilities according to their needs. They offer merchants the ability to craft a tailored B2B ecommerce experience for their business partners while also allowing agencies to fine-tune Shopware to meet specific requirements. This means that B2B Components can be individually activated or deactivated for each business partner within the shop.
```

Comment:
There are lot of synonyms used. Do we call it as B2B Components / B2B features / B2B Capabilities across the docs ?

Reply:
Mostly I only see using B2B components for all relevant cases. As a way to differentiate from B2B suite. Sometimes we use B2B features but only for discuss technical purposes.

Therefore, in my opinion, we should only use B2B components for consistency